pkg/resource/grafana-datasource/grafana-loki-datasource.go

```diff
@@ -0,0 +1,111 @@
+package grafanadatasource
+
+import (
+	"encoding/base64"
+
+	"github.com/giantswarm/logging-operator/pkg/common"
+	loggedcluster "github.com/giantswarm/logging-operator/pkg/logged-cluster"
+	loggingcredentials "github.com/giantswarm/logging-operator/pkg/resource/logging-credentials"
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	lokiURL                   = "http://loki-gateway.loki.svc.cluster.local"
+	datasourceName            = "Loki"
+	datasourceSecretName      = "loki-datasource"
+	datasourceSecretNamespace = "monitoring"
```

Comment:
Is there a reason why we create this in the monitoring namespace? We could add it in the loki one right?

Reply:
Yes, that's arbitrary; could be in any namespace.